vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue

```diff
@@ -172,22 +195,26 @@ export default {
     totalBedrooms: ['1', '2', '3', '4', '5'],
     totalBaths: ['1', '1.5', '2', '2.5', '3', '3.5'],
     properties: ["Entire House", "Full Apartment", "Private Room",],
-    states: [
-      'Alabama', 'Alaska', 'American Samoa', 'Arizona',
-      'Arkansas', 'California', 'Colorado', 'Connecticut',
-      'Delaware', 'District of Columbia', 'Federated States of Micronesia',
-      'Florida', 'Georgia', 'Guam', 'Hawaii', 'Idaho',
-      'Illinois', 'Indiana', 'Iowa', 'Kansas', 'Kentucky',
-      'Louisiana', 'Maine', 'Marshall Islands', 'Maryland',
-      'Massachusetts', 'Michigan', 'Minnesota', 'Mississippi',
-      'Missouri', 'Montana', 'Nebraska', 'Nevada',
-      'New Hampshire', 'New Jersey', 'New Mexico', 'New York',
-      'North Carolina', 'North Dakota', 'Northern Mariana Islands', 'Ohio',
-      'Oklahoma', 'Oregon', 'Palau', 'Pennsylvania', 'Puerto Rico',
-      'Rhode Island', 'South Carolina', 'South Dakota', 'Tennessee',
-      'Texas', 'Utah', 'Vermont', 'Virgin Island', 'Virginia',
-      'Washington', 'West Virginia', 'Wisconsin', 'Wyoming',
+    titleRules: [
+      v => !!v || 'A Listing Title is Required',
```

Comment:
Can we perhaps use better variable naming here? It's not easily identifiable to someone with no context reading this code what exactly `v` is referring to. Perhaps also a comment on what `!!v` is? Because looking at it logically `!!v == v` so at first it doesn't make sense. It wasn't until I looked it up that I saw it also seems to be a typecast operator?

Reply:
Yeah definitely. It'll probably be cleaner if I change the naming.